src/icons.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { useId } from '@reach/auto-id';
+
+export const IconMinor: React.FC<Omit<
```

Comment:
Matter of preference, but I would create a single `Icon` component and have it take a prop. But, we can do this later if we'll start having more icons 🙃